src/identity/components/MarketoAccountUpgradeOverlay.tsx

```diff
@@ -137,9 +161,19 @@ export const MarketoAccountUpgradeOverlay: FC = () => {
     try {
       window.MktoForms2.getForm(MARKETO_FORM_ID).submit()
       dispatch(notify(marketoFormSubmitSuccess()))
+      event(AccountUpgradeOverlay.MarketoAccountUpgradeSuccess, multiOrgTag, {
+        userId: user.id,
+        accountId,
+        accountType,
+      })
       onClose()
     } catch (err) {
       handleError(MarketoError.FormSubmitError, err)
+      event(AccountUpgradeOverlay.MarketoAccountUpgradeFail, multiOrgTag, {
+        userId: user.id,
+        accountId,
+        accountType,
+      })
     }
```

Comment:
I actually don't think we need the data in the `catch` evented - an upgrade failing isn't an event we can generate user behavior data from, so much as an indicator of a problem in prod that we'd need to catch and resolve. The existing handler sends that data to honeybadger, which I think is all we need to do here.

If you remove the event from the catch, I think we could just send the success event to analytics as `MarketoAccountUpgrade`, which I suspect is the data that would be useful - how many users are upgrading accounts; when are they doing that; etc.

Reply:
ah that's a fair point. I'll remove it